esp32/modesp.c

```diff
@@ -78,6 +79,16 @@ STATIC mp_obj_t esp_flash_user_start(void) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_0(esp_flash_user_start_obj, esp_flash_user_start);
 
+STATIC mp_obj_t esp_neopixel_write_(mp_obj_t pin, mp_obj_t buf, mp_obj_t is800k) {
```

Comment:
Probably better to make the is800k parameter a generic "timing" parameter.  Eg a value of 0 is 400khz, 1 is 800khz, 2 could be APA (if it differs), etc.